internal/store/postgres/migrations/000021_alter_asset_table_is_deleted.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE assets ADD COLUMN IF NOT EXISTS is_deleted BOOLEAN DEFAULT false;
```

Comment:
would be better if we can use `deleted_at` with type timestamp and default null. At least it will give us also the information of when it was deleted.

I'm ok with this approach if changing it to `deleted_at` would be a big changes

Reply:
JFYI this is the rationale of the decision https://go-jek.atlassian.net/wiki/spaces/DE/pages/3473802271/COM-RFC-24008+-+Soft+Deletion

Reply:
yes, also for deletion date information will be saved on `updated_at` in respective `assets_versions` table
This will also saved restoration date